src/View/BakeView.php

```diff
@@ -45,7 +45,8 @@ class BakeView extends TwigView
     public function initialize(): void
     {
         $this->setConfig('environment', [
-          'cache' => false,
+        'autoescape' => false,
```

Comment:
Indentation looks off, surprising the phpcs didn't catch ti.

Reply:
These were not indented 4 spaces before for some reason which messed up my editor.